decanter.gemspec

```diff
@@ -20,19 +22,17 @@ Gem::Specification.new do |spec|
 
   spec.metadata['allowed_push_host'] = 'https://rubygems.org'
 
-  spec.files         = `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
+  spec.files = `git ls-files -z`.split("\x0").reject do |f|
+    f.match(%r{^(test|spec|features)/})
+  end
   spec.bindir        = 'exe'
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ['lib']
 
-  spec.add_dependency 'rails', '>= 7.1.3.2'
   spec.add_dependency 'actionpack', '>= 7.1.3.2'
   spec.add_dependency 'activesupport'
+  spec.add_dependency 'rails', '>= 7.1.3.2'
   spec.add_dependency 'rails-html-sanitizer', '>= 1.0.4'
 
-  spec.add_development_dependency 'bundler', '~> 2.4.22'
-  spec.add_development_dependency 'dotenv'
-  spec.add_development_dependency 'rake', '~> 12.0'
-  spec.add_development_dependency 'rspec-rails', '~> 3.9'
-  spec.add_development_dependency 'simplecov', '~> 0.15.1'
+  spec.metadata['rubygems_mfa_required'] = 'true'
```

Comment:
@nicoledow not sure if MFA is already required for publishing this gem but rubocop is requiring it. Seems like a good idea.

Ref: https://guides.rubygems.org/mfa-requirement-opt-in/